torchtext/modules/multiheadattention.py

```diff
@@ -0,0 +1,229 @@
+import torch
+from typing import Tuple, Optional
+
+
+class MultiheadAttentionContainer(torch.nn.Module):
```

Comment:
If initializes is forward setup such that an instance of this could be a drop-in replacement of an instance of torch.nn.MHA?